docs/ui.rst

```diff
@@ -0,0 +1,94 @@
+==============
+User Interface
+==============
+
+Thebe includes support for some built-in user interface elements.
+
+The aim is to provide simple elements that can be added to a page a styled accordingly by the consumer without neeing to [re]write coont code.
+Using these UI elements in the way shown here, assumes that you already have Thebe statically loaded on page.
+
+Built in UI elements are:
+
+- Activate Button
+- Kernel Status Widget
+
+Activate Button
+===============
+
+Configure this UI element as follows.
+
+Add a :code:`div` element to the page in the desired location.
+
+.. code-block:: html
+
+  <div class="thebe-activate" />
+
+Then set the following option to :code:`mountActivateWidget:true` in the Thebe config object
+
+Status Widget
+=============
+
+Configure this UI element as follows.
```

Comment:
```suggestion
This element will display the status of the kernel, it will update when the kernel sends new messages or when code is executed.
Configure this UI element as follows.
```